### ISSUE TITLE
[Bug]: Headless Browser (new mode) Displays Native Dropdown List on Select Element Click

### ISSUE THREAD

**Issue by almog-geva** [puppeteer/puppeteer#10198]
### Bug expectation

I expect that when the `headless` option is set to `'new'` in Puppeteer's `launch` method, the native dropdown list UI should not be displayed upon clicking a  `<select>` element. This is the typical behavior of a headless browser where user interface elements are not rendered or displayed.

However, what is happening instead is that, upon clicking a `<select>` element, the browser's native dropdown list UI is consistently displayed. This causes significant disruption to the automation process, as the click operation does not complete until the dropdown list is closed, either manually or programmatically. Notably, this issue does not occur when `headless` is set to `true`.

https://github.com/puppeteer/puppeteer/assets/16611980/0ab15ba5-3829-4de0-8151-aabef8ad6c2f

### Bug behavior

- [ ] Flaky
- [ ] PDF

### Minimal, reproducible example

```TypeScript
import puppeteer from 'puppeteer';

(async () => {
  const browser = await puppeteer.launch({headless: 'new'});

  const [page] = await browser.pages();

  await page.goto('https://www.ebay.com/');

  const selectElement = await page.waitForSelector('aria/Select a category for search');

  await selectElement?.click();

  await page.close();
  await browser.close();
})();
```

### Error string

no error

### Puppeteer version

20.2.1

### Node version

18.12.1

### Package manager

npm

### Package manager version

9.5.1

### Operating system

macOS

**Comment by OrKoN** [comment]
https://bugs.chromium.org/p/chromium/issues/detail?id=1446277

**Comment by ruslanx3m** [comment]
Here how I fixed, piece of code:


  async crashFix() {

    if (!this._crashFixing) return;

    const crashFile = join(this._userDataDir, 'Default', 'Preferences');

    const fileExists = await fs.pathExists(crashFile);

    if (!fileExists) return;

    try {

      const crashFileContent = await fs.readFile(crashFile, 'utf8');

      if (crashFileContent.includes('"exit_type":"Crashed"')) {

        // set "exit_type":"Crashed" to "exit_type":"Normal"
        const newCrashFileContent = crashFileContent.replace('"exit_type":"Crashed"', '"exit_type":"Normal"');

        // write new crash file to disk
        await fs.writeFile(crashFile, newCrashFileContent, 'utf8');
      }

    } catch (error) {
      // console.log(error);
    }
  }

**Comment by OrKoN** [comment]
It should be fixed with M116 (pptr v 21.1.0).
